webpack.prod.config.js

```diff
@@ -17,6 +19,7 @@ module.exports = {
     new UglifyJSPlugin({
       sourceMap: true,
     }),
+    new ExtractTextPlugin('styles.css'),
```

Comment:
This seems like its setup for a single stylesheet. Did you double check that this setup works for module sheets?

Reply:
I'm confused.  I thought this plugin takes automatically separate css modules as as input, and bundles them into a single stylesheet as an output?  So in our case, the output is styles.css.

The documentation expects a single filename as an argument:

https://github.com/webpack-contrib/extract-text-webpack-plugin#options

Reply:
That's my question. Just wanted to confirm its doing that since we're going to be using more than one stylesheet. I looked at some references in their docs that allow for params to have multiple style sheets using something like array notation or object destructuring